doc/releases/release-notes-3.1.rst

```diff
@@ -1123,3 +1123,799 @@ Issue Related Items
+* :github:`46138` - Problem with building zephyr/samples/subsys/mgmt/mcumgr/smp_svr  using atsame70
+* :github:`46137` - RFC: Integrate u8g2 monochrome graphcial library as module to Zephyr OS (https://github.com/olikraus/u8g2)
+* :github:`46129` - net: lwm2m: Object Update Callbacks
+* :github:`46102` - samples: net: W5500 implementation
+* :github:`46097` - b_l072z_lrwan1 usart dma doesn't work
+* :github:`46093` - get a run error "Fatal exception (28): LoadProhibited" while enable CONFIG_NEWLIB_LIBC=y
+* :github:`46091` - samples: net: cloud: tagoio: Drop pinmux dependency
+* :github:`46059` - LwM2M: Software management URI resource not updated properly
+* :github:`46056` - ``unexpected eof`` with twister running ``tests/subsys/logging/log_api/logging.log2_api_immediate_printk_cpp`` on ``qemu_leon3``
+* :github:`46037` - ESP32 :  fails to build the mcuboot, zephyr v3.1.0 rc2,  sdk 0.14.2
+* :github:`46034` - subsys settings: should check the return value of function cs->cs_itf->csi_load(cs, &arg).
+* :github:`46033` - twister: incorrect display of test results
+* :github:`46027` - tests: rpi_pico tests fails on twister with: No rule to make target 'bootloader/boot_stage2.S
+* :github:`46026` - Bluetooth: Controller: llcp: Wrong effective time calculation if PHY changed
+* :github:`46023` - drivers: reset: Use of reserved identifier ``assert``
+* :github:`46020` - module/mcuboot: doesn't build with either RSA or ECISE-X25519 image encryption
+* :github:`46017` - Apply for contributor
+* :github:`46002` - NMP timeout when i am using  any mcumgr command
+* :github:`45996` - stm32F7: DCache configuration is not correctly implemented
+* :github:`45948` - net: socket: dtls: sendmsg() should result in single outgoing datagram
+* :github:`45946` - net: context: outgoing datagrams are truncated if not enough memory was allocated
+* :github:`45942` - tests: twister: harness: Test harness report pass when there is no console output
+* :github:`45933` - webusb sample code linking error for esp32 board
+* :github:`45932` - tests: subsys/logging/log_syst : failed to build on rpi_pico
+* :github:`45916` - USART on STM32: Using same name for different remapping configurations
+* :github:`45911` - LVGL sample cannot be built with CONFIG_LEGACY_INCLUDE_PATH=n
+* :github:`45904` - All tests require full timeout period to pass after twister overhaul when executed on HW platform
+* :github:`45894` - up_squared：the test shows pass in the twister.log it but does not seem to finish
+* :github:`45893` - MCUboot authentication failure with RSA-3072 key on i.MX RT 1160 EVK
+* :github:`45886` - ESP32: PWM parameter renaming broke compilation
+* :github:`45883` - Bluetooth: Controller: CCM reads data before Radio stores them when DF enabled on PHY 1M
+* :github:`45882` - Zephyr minimal C library contains files licensed with BSD-4-Clause-UC
+* :github:`45878` - doc: release: Update release notes with CVE
+* :github:`45877` - version bump to 2.7.1
```

Comment:
There are a lot of 2.7 branch details in here.  Are they needed for 3.1 release notes?